tox.ini

```diff
@@ -98,12 +98,12 @@ commands =
     pyright: pyright {posargs}
     mypy: mypy --install-types --non-interactive {posargs:.}
 
-[testenv:format-{black,ruff,codespell}]
+[testenv:format-{ruff,codespell}]
 description = Automatically format source code
 base = testenv, lint
 labels = format
 commands =
-    black: black {tty:--color} {posargs} .
+    ruff: ruff format {posargs} .
```

Comment:
Above you use `{posargs:.}` and here you're going with `{posargs} .`, shouldn't it be aligned ?

Reply:
Well spotted. Adjusted

Reply:
As a sidenote, `--respect-gitignore` is now default so I removed that too.